app/views/embed/topics.html.erb

```diff
@@ -1,4 +1,12 @@
 <%- if @list && @list.topics.present? %>
+  <%- if @allow_create %>
+    <%= link_to @create_url, target: "_blank" do |link| %>
+      <button class=<%="new-topic-btn" + (@template == "complete" ? " complete" : "") %>>
+        <span class=<%="new-topic-btn__icon" if @template == "complete" %>></span>
+        <span class=<%="new-topic-btn__text" if @template == "complete" %>>Create New Topic!</span>
```

Comment:
This needs to be in the server i18n file.

Also the standard for Discourse is to not use an exclamation point